src/cdpTargetsProvider.ts

```diff
@@ -107,7 +107,7 @@ export class CDPTargetsProvider implements vscode.TreeDataProvider<CDPTarget> {
         }
         const faviconRegex = /((?:\/\/|\.)([^\.]*)\.[^\.^\/]+\/).*/;
 
-        // Example regex match: https://docs.microsoft.com/en-us/microsoft-edge/
+        // Example regex match: https://learn.microsoft.com/en-us/microsoft-edge/
```

Comment:
@codepo8 - retained "/en-us" bc also on next line

Reply:
You can remove the en-us without a problem, this makes no difference to the regex here.